### PR TITLE
Migration from JCenter to mavenCentral

### PR DESCRIPTION
**Pull request by efguydan**
This Pull Request removes dependency on jCenter() and migrates to releasing artifacts to mavenCentral().

### CODE REVIEW

**Comment by rasheedsulayman** [comment]
Please squash the commits to one when you want to merge. You can see it in the drop down beside "Merge pull request"

File: publish/publish_module.gradle [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    description = 'A library to dynamically format your EditTexts to take currency inputs.'
+                    url = 'https://github.com/CottaCush/CurrencyEditText'
+                    licenses {
+                        license {
+                            name = 'Apache License'
+                            url = 'https://github.com/CottaCush/CurrencyEditText/blob/master/LICENSE.md'
+                        }
+                    }
+                    developers {
+                        developer {
```
Comment (rasheedsulayman) [inline publish/publish_module.gradle]: Do we need to hardcode contributor names?
Is it possible for it to be pulled from the commit history?
Or do we even need to add the information at all?

Comment (efguydan) [reply publish/publish_module.gradle]: Hmm. On being pulled from commit history, I can check if that's possible. 
On whether the information actually needs to be added, I don't think its necessary. 
Do you think we shouldn't add it if it isn't absolutely necessary?

Comment (rasheedsulayman) [reply publish/publish_module.gradle]: Don't add it. I mean we have other contributors as well.

Comment (efguydan) [reply publish/publish_module.gradle]: Alright. Sounds good.

**Review by rasheedsulayman — APPROVED** [review]

**Pull request merged**